content/plugins/overview.md

```diff
@@ -17,60 +17,9 @@ To cover the two major types of plugins (IaaS and Management tools), let's also
 
 # Plugin Development
 
-You should check out the [Plugins Authoring Guide](plugins-authoring.html) if you want to write your own plugin for your chosen tool or IaaS provider.
+You should check out the [Plugins Authoring Guide]({{< relref "plugins/creating-your-own-plugin.md" >}}) if you want to write your own plugin for your chosen tool or IaaS provider.
```

Comment:
Please don't call it the "Plugin Authoring Guide" :)
Maybe: You should check out the "Creating your own plugin" page if...
?